### ISSUE TITLE
Coronavirus can travel twice as far as official ‘safe distance’ and stay in air for 30 minutes

### ISSUE THREAD

**Issue by codejack42** [flattenthecurve/guide#225]
**Content to be added or changed:**

**tl;dr:** _Contrary to current expert guidance: According to a study by a team of Chinese government epidemiologists, published in peer-review journal "Practical Preventive Medicine", the coronavirus that causes Covid-19 can linger in the air for at least 30 minutes, travel up to 4.5 metres, and be prevented by the potential target wearing a simple mask. Study's authors advise that everyone wear a mask in closed environment such as a bus, and for the entire duration of any bus rides. Study warns that the coronavirus can survive more than five days in human faeces or bodily fluids. Problematic since current expert guidance indicates that 1-2 meters of space between people is adequate, that aerosols are inactive in under 30 minutes, that simple masks are generally not helpful for potential targets, and that faeces and bodily fluids need not be treated as a significant factor in transmission. This information is potentially important for people who are self-isolating together as well as for those caring for someone in the home and for community medics and clinicians making diagnoses while trying to avoid infection. Also potentially important for those undertaking more public activities such shopping for food and other necessities. Bottom line is that this casts a lot of doubt on current conventional wisdom. Maybe not enough doubt to merit a drastic change in the Guide but maybe enough to merit minor changes to language such as "2-5 meters, erring toward 5 when possible". And in any case we should be on the lookout for further developments in the areas examined in the "Practical Preventive Medicine" paper._

Someone with more knowledge than I should look at the study and see what kinds of response and/or corroborating evidence there is.

Article in _South China Morning Post_:
**"Coronavirus can travel twice as far as official ‘safe distance’ and stay in air for 30 minutes, Chinese study finds"**
https://www.scmp.com/news/china/science/article/3074351/coronavirus-can-travel-twice-far-official-safe-distance-and-stay

++++ BEGIN HACKISH ABSTRACT ++++
A single closely-studied and tracked mildly symptomatic patient in China (Patient "A"), in the course of an ordinary 4-hour intercity bus ride infected 7 people, some of whom were sat 4.5 meters from "A". At the end of the ride "A" and all others disembarked, the bus sat empty for 30 minutes before the next group boarded, and one of the next group boarding also became infected as a result of "A" having ridden the bus 30 minutes prior. "A" next boarded a minibus for a 2-hour trip, during which "A" infected another 2 people, one of whom was also sat 4.5 meters from "A". This is a total of 10 people. Of additional note, "A" was not wearing a mask nor were any of those who became infected by "A". Among the passengers who were wearing a mask, none were infected by "A".
++++ END HACKISH ABSTRACT ++++

++++ BEGIN EXCERPTS ++++
The coronavirus that causes Covid-19 can linger in the air for at least 30 minutes and travel up to 4.5 metres – further than the “safe distance” advised by health authorities around the world, according to a study by a team of Chinese government epidemiologists.
...
China requires closed circuit television cameras to be installed on all long-distance buses, which provided valuable footage for researchers to reconstruct the spread of the virus on the bus, whose windows were all closed.
...
“It can be confirmed that in a closed environment with air-conditioning, the transmission distance of the new coronavirus will exceed the commonly recognised safe distance,”
...
The scientists warned that the coronavirus could survive more than five days in human faeces or bodily fluids.
...
... the study proves the importance of washing hands and wearing face masks in public places because the virus can linger in the air attached to fine droplet particles.
...
Our advice is to wear a face mask all the way [through the bus ride],” they added.
...
...security camera footage showed patient “A” did not interact with others throughout the four-hour ride.
...
“The possible reason is that in a completely enclosed space, the airflow is mainly driven by the hot air generated by the air conditioning. The rise of the hot air can transport the virus-laden droplets to a greater distance,”
...
By the time the study was finished in mid February, patient “A” had infected at least 13 people.
...
It is generally believed that the airborne transmission of Covid-19 is limited because the tiny droplets produced by patients will quickly sink to the ground. This belief has prompted the Chinese health authorities to suggest that people should stay a metre apart in public and the US Centres for Disease Control recommend a safe distance of six feet (about 1.8 metres).
...
The researchers also found that none of those passengers in the two buses who wore face masks were infected.
...
“When riding on more closed public transportation such as subways, cars, planes, etc, you should wear a mask all the time, and at the same time, minimise the contact between your hands and public areas, and avoid touching your face before cleaning,” they said.
...
The researchers also suggested improving sanitation on public transport and adjusting the air conditioning to maximise the volume of fresh air supplied.
++++ END EXCERPTS ++++

**Comment by jmcmurry** [comment]
When you say "survive in feces" do you mean they recovered viable virus, or that this was still RNA+?
